### PR TITLE
1st pull - need review

### PR DESCRIPTION
**Pull request by shakabandai**

### CODE REVIEW

**Comment by underyourcover** [comment]
looks great to me, this code is 🔥

File: client/index.jsx [hunk 1]
```diff
@@ -3,21 +3,29 @@ import ReactDOM from 'react-dom';
 import Sources from './src/components/sources.jsx';
 import axios from 'axios';
 
+
 class App extends React.Component {
-    constructor(props){
+    constructor(props) {
```
Comment (therobinkim) [inline client/index.jsx]: use 2 spaces for indenting, not 4

File: server/index.js [hunk 2]
```diff
@@ -6,31 +6,32 @@ const bodyParser = require('body-parser');
...
-  
-  let port = 3001;
-  
-  app.listen(port, () => {
-    console.log(`listening on port ${port}`);
-  });
-  
\ No newline at end of file
+  let user = req.body;
+  console.log('posting at server: '.req.body);
```
Comment (therobinkim) [inline server/index.js]: this looks like a syntax error

File: client/index.jsx (hunk 1 above)
Comment (shakabandai) [reply client/index.jsx]: right; i think it did that automatically with vscodes settings. updating them to airbnb specs.
thanks for the catch.

**Pull request merged**